net-p2p/deluge/deluge-2.1.1.ebuild

```diff
@@ -0,0 +1,169 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_SINGLE_IMPL=1
+inherit xdg distutils-r1 systemd
+
+DESCRIPTION="BitTorrent client with a client/server model"
+HOMEPAGE="https://deluge-torrent.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.deluge-torrent.org/${PN}"
+else
+	SRC_URI="http://download.deluge-torrent.org/source/$(ver_cut 1-2)/${P}.tar.xz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="console gui libnotify sound webinterface"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	libnotify? ( gui )
+	sound? ( gui )
+"
+
+BDEPEND="
+	dev-util/intltool
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/pytest-twisted[${PYTHON_USEDEP}]
+		')
+	)
+"
+
+RDEPEND="
+	acct-group/deluge
```

Comment:
@Kangie , happy new year!

`acct-user/deluge` has [already](https://packages.gentoo.org/packages/acct-user/deluge/dependencies) `RDEPEND`'ed `acct-group/deluge`.

So, we can delete this dependency. (And as I see, other packages don't have such.)

What do you think?

Reply:
Addressed in #28956 .